src/status_im/contexts/wallet/add_account/add_address/view.cljs

```diff
@@ -109,22 +123,27 @@
 
 (defn view
   []
-  (let [addresses           (rf/sub [:wallet/addresses])
-        input-value         (reagent/atom nil)
-        validate            #(validate-address (set addresses) %)
-        validation-msg      (reagent/atom nil)
-        clear-input         (fn []
-                              (reset! input-value nil)
-                              (reset! validation-msg nil)
-                              (rf/dispatch [:wallet/clear-address-activity])
-                              (rf/dispatch [:wallet/clean-scanned-address]))
-        customization-color (rf/sub [:profile/customization-color])]
+  (let [addresses            (rf/sub [:wallet/addresses])
+        lowercased-addresses (map string/lower-case addresses)
```

Comment:
looks like probably you want this int the sub since you are not using `addresses`?